src/helpers/sync_helpers.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Helper functions for syncing branches
+
+
+# Outputs the steps to sync the given branch
+function sync_branch_steps {
+  local branch=$1
+  local is_feature ; is_feature="$(is_feature_branch "$branch")"
```

Comment:
That's a new check in ShellCheck 0.3.6: https://github.com/koalaman/shellcheck/wiki/SC2155